tasmota/xdrv_03_energy.ino

```diff
@@ -1108,7 +1108,7 @@ void EnergyShow(bool json) {
       EnergyFormatSum(value2_chr, Energy.daily, Settings->flag2.energy_resolution, json));
 
 /*
- #if defined(SDM630_IMPORT) || defined(SDM72_IMPEXP)
+ #if defined(SDM630_IMPORT) || defined(SDM72_IMPEXP) 
```

Comment:
Don't touch unchanged files globbering the PR. Remove spaces at end of line.

Reply:
removed space